### PR TITLE
Center align homepage items

### PR DESCRIPTION
**Pull request by chasulin**

### CODE REVIEW

File: src/components/Explore/Explore.tsx [hunk 1]
```diff
@@ -50,8 +50,7 @@ import {
 import { ScreenshotReady } from 'components/Screenshot';
 import { EventCategory, EventAction, trackEvent } from 'components/Analytics';
 import regions, { Region, useRegionFromParams } from 'common/regions';
-import { SectionHeader } from 'components/SharedComponents';
-import NationalText from 'components/NationalText';
+// import { SectionHeader } from 'components/SharedComponents';
```
Comment (mikelehen) [inline src/components/Explore/Explore.tsx:53]: remove?

File: src/screens/HomePage/HomePage.style.tsx [hunk 2]
```diff
@@ -106,7 +106,7 @@ export const SectionWrapper = styled.div`
   border-top: 1px solid ${props => props.theme.palette.lightGray};
 `;
 
-export const SectionHeader = styled(Subtitle1)`
+export const SmallSectionHeader = styled(Subtitle1)`
```
Comment (mikelehen) [inline src/screens/HomePage/HomePage.style.tsx:109]: 🙏  for renaming!

File: src/components/Compare/Filters.tsx [hunk 3]
```diff
@@ -28,9 +28,9 @@ const Filters = (props: {
   const { setHomepageScope } = props;
 
   const homepageFilterLabels = [
+    homepageLabelMap[HomepageLocationScope.STATE].plural,
```
Comment (mikelehen) [inline src/components/Compare/Filters.tsx:31]: Oooh, nice. This was kinda' bugging me too. 😁

**Review by mikelehen — APPROVED** [review]
Sweet, looks good!

File: src/components/Explore/Explore.tsx (hunk 1 above)
Comment (chasulin) [reply src/components/Explore/Explore.tsx:53]: oh whoops yes

**Pull request merged**